README.md

```diff
@@ -31,7 +31,7 @@ Currently, the supported banks are:
 You can install this package using pip:
 
 ```bash
-pip install israel-bank-account-validator
+pip install git+https://github.com/republixapps/israel-bank-account-validator.git@rc#egg=israel_bank_account_validator
```

Comment:
how we will not forget drop RC version when it will be merged to master?

Reply:
It's decide of Kirll, to take always from RC, 
I just updated the README